medusa-config.js

```diff
@@ -1,45 +1,45 @@
-// CORS when consuming Medusa from admin
-const ADMIN_CORS = process.env.ADMIN_CORS || "http://localhost:7000,http://localhost:7001";
+import * as dotenv from 'dotenv';
```

Comment:
Are the changes in this file necessary? I think it's ok to keep them as it is installed with Medusa

Reply:
Medusa only load .env file which does not allow to set different env depending on the execution environment.

Reply:
Also, the config file is the only place that calls the dotenv package. That config is called in two places during the loading, expressLoader and passport which are both called later during the loading flow. It ensure that at any time the medusa-config is properly populated no matter the environment. If I am missing something, please point it out to me :)

Reply:
Ok I don't mind making this change but is it necessary to mention in the tutorial? Does it play part in any of the other parts?